packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-styles.js

```diff
@@ -34,12 +34,28 @@ const multiSelectComboBox = css`
     caret-color: var(--lumo-body-text-color) !important;
   }
 
-  [part~='chip']:not(:last-of-type) {
+  /* Override input-container styles */
+  ::slotted([slot='chip']),
+  ::slotted([slot='overflow']) {
+    min-height: auto;
+    padding: 0.3125em calc(0.5em + var(--lumo-border-radius-s) / 4);
+    color: var(--lumo-body-text-color);
+    -webkit-mask-image: none;
+    mask-image: none;
+  }
```

Comment:
These styles are needed to override this CSS:

https://github.com/vaadin/web-components/blob/6e04c8dbfdaf839275441cd6374b399aa0dcf5b9/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L37-L44

Reply:
This is a bit unfortunate, I was confused at first why the focus styles below are not defined in the chip styles. But probably a larger effort to resolve this in the input container styles.

Anyway, visual tests seem to cover this, so it's fine for now.

Reply:
When I originally created `vaadin-input-container` for Vaadin 22, this use case wasn't taken into account.
We might need to change it, to avoid making such overrides every now and then. I'll create an issue.